src/_gettsim/transfers/wohngeld.py

```diff
@@ -657,3 +657,40 @@ def wohngeld_vor_vermög_check_m_hh(
         )
 
     return out
+
+
+def kinderwohngeld_m(
```

Comment:
Should again probably contain `vor_vorrang_check`

Reply:
If anything, it's `nach_vorrang_check`

Reply:
Why? it uses the input `wohngeld_vor_vorrang_check_m`? And it represents potential Kinderwohngeld not the Kindergeld that is indeed paid out, right?

Reply:
Because of `eigenbedarf_gedeckt` the outcome of the Vorrangprüfung is already specified. But given your comment above, we might get rid of this function anyway.